articles/virtual-desktop/start-virtual-machine-connect-faq.md

```diff
@@ -28,11 +28,11 @@ To configure the deallocation policy:
 >[!NOTE]
 >Make sure to set the time limit for the "End a disconnected session" policy to a value greater than five minutes. A low time limit can cause users' sessions to end if their network loses connection for too long, resulting in lost work.
 
-Signing users out won't deallocate their VMs. To learn how to deallocate VMs, see [Start or stop VMs during off hours](../automation/automation-solution-vm-management.md).
+Signing users out won't deallocate their VMs. To learn how to deallocate VMs, see [Start or stop VMs during off hours](../automation/automation-solution-vm-management.md) for personal host pools and [Scale session hosts using Azure Automation](../set-up-scaling-script.md) for pooled host pools.
```

Comment:
@jamasten Could you change the link to `[Scale session hosts using Azure Automation](set-up-scaling-script.md)`? The file you're linking to is in the [/virtual-desktop/ folder](https://github.com/MicrosoftDocs/azure-docs-pr/tree/master/articles/virtual-desktop), so omit the `../` for the link. 

Usually, I can make this change, but the **Edit file** button is inactive for me in this pull request. Thanks.